src/collections/_documentation/development/sdk-dev/overview.md

```diff
@@ -77,7 +77,7 @@ Sentry.init({
 ```
 
 {% capture __alert_content -%}
-If an empty DSN is passed, you should treat it as valid option which signifies disabling the SDK.
+It is valid to configure the SDK with an empty DSN.
```

Comment:
Suggest "Configure the SDK so that an empty DSN is a valid option"

Q - does this disable the SDK? If so, that is not captured in the new content (nor my suggested edit)